recipes/minepy/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "minepy" %}
+{% set version = "1.2.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "04d9525a353065159d6f86d6c70f33b738d437b53a41527462cbe719225c25ed"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - numpy >=1.3.0
+    - pip
+    - python
+  run:
+    - numpy >=1.3.0
```

If you are compiling against ``numpy`` you should use
```suggestion
    - {{ pin_compatible('numpy') }}
```